langlib/lang.query/src/main/ballerina/helpers.bal

```diff
@@ -181,3 +215,18 @@ function consumeStream(stream<Type, CompletionType> strm) returns any|error {
         return v;
     }
 }
+
+function createImmutableTable(table<map<Type>> tbl, Type[] arr) returns table<map<Type>> & readonly = @java:Method {
+    'class: "org.ballerinalang.langlib.query.CreateImmutableType",
+    name: "createImmutableTable"
+} external;
+
+function createTableWithKeySpecifier(table<map<Type>> tbl, typedesc tableType) returns table<map<Type>> = @java:Method {
+    'class: "org.ballerinalang.langlib.query.CreateImmutableType",
+    name: "createTableWithKeySpecifier"
+} external;
+
+function createImmutableType(any mutableType) = @java:Method {
+    'class: "org.ballerinalang.langlib.query.CreateImmutableType",
+    name: "createImmutableType"
```

Comment:
Did you check if we can map this to `io.ballerina.runtime.api.values.BRefValue#freezeDirect` instead?

Reply:
Yeah. Couldn't find a way to pass the arguement. Is there anyway to pass the ballerina variable to `io.ballerina.runtime.api.values.BRefValue#freezeDirect` method?

Reply:
Seems like instance methods require the argument to be handle - https://ballerina.io/learn/java-interoperability/ballerina-ffi/#call-instance-methods. NVM, let's go with the current approach.